dtfs-central-api/src/services/state-machines/fee-record/fee-record.state-machine.ts

```diff
@@ -64,6 +70,10 @@ export class FeeRecordStateMachine {
             return handleFeeRecordPaymentDeletedEvent(this.feeRecord, event.payload);
           case 'PAYMENT_EDITED':
             return handleFeeRecordPaymentEditedEvent(this.feeRecord, event.payload);
+          case 'REMOVE_FROM_PAYMENT':
```

Comment:
`REMOVE_FROM_PAYMENTS` or perhaps `REMOVE_FROM_PAYMENT_GROUP`?

Reply:
Changed to the latter - `REMOVE_FROM_PAYMENT_GROUP` - in commit 65ee68db8fe709731880bdd096c86b31cf00e11c thanks